src/ansys/pre_commit_hooks/tech_review.py

```diff
@@ -0,0 +1,253 @@
+
+    return non_compliant
+
+
+def generate_file_from_jinja(file, project_name, year_str):
+    """Generate file using jinja templates."""
+    loader = FileSystemLoader(searchpath=os.path.join(HOOK_PATH, "templates"))
+    env = Environment(loader=loader)
+    template = env.get_template(file)
+    file_content = template.render(
+        doc_repo_name=DOC_REPO_NAME, project_name=project_name, year_span=year_str
+    )
+
+    return file_content
+
+
+def check_file_content(file, generated_content):
+    """Check file content."""
+    non_compliant = 0
+
+    generated_file = NamedTemporaryFile(mode="w", delete=False).name
+    with open(generated_file, "w") as f:
+        f.write(generated_content)
+
+    # Check if CONTRIBUTORS.md content has been changed from template
+    if file == "CONTRIBUTORS.md" and check_same_content(file, generated_file):
+        print("Please update your CONTRIBUTORS.md file.")
+        non_compliant = 1
+    # Check if the license phrase is in LICENSE (by default, MIT)
+    elif file == "LICENSE":
+        with open(file, "r") as f:
+            if "MIT" not in f.readline():
+                print(f"MIT is not in {file}")
+
+    return non_compliant
+
+
+# check there are src/ tests/ doc/ folders in root of git repo
+# verify there are no dockerfiles at the root level - should be in doc folder
+# check there's a dependabot.yml file
+
+# eventually - check the ci_cd yml file actions doc-style, code-style, etc
+# run pytest, parse coverage files
+
+
+def main():
+    """Check files for technical review."""
+    parser = argparse.ArgumentParser()
+    # Get list of committed files
+    parser.add_argument("files", nargs="*")
+    # Get the name of the authors and maintainers of the project
+    parser.add_argument(
+        "--author_maint_name",
+        type=str,
+        help="Name of the authors and maintainers of the project.",
+        default=DEFAULT_AUTHOR_MAINT_NAME,
+    )
+    # Get the email of the authors and maintainers of the project
+    parser.add_argument(
+        "--author_maint_email",
+        type=str,
+        help="Email of the authors and maintainers of the project.",
+        default=DEFAULT_AUTHOR_MAINT_EMAIL,
+    )
+    # Get the start year of the repository
+    parser.add_argument(
+        "--start_year", type=str, help="Start year of the repository.", default=DEFAULT_START_YEAR
+    )
+    # Get the license the repository uses
+    parser.add_argument(
+        "--license", type=str, help="License that the repository uses.", default=DEFAULT_LICENSE
+    )
+
+    count = 0
+    args = parser.parse_args()
+    author_maint_name = args.author_maint_name
+    author_maint_email = args.author_maint_email
+    start_year = args.start_year
+
+    check_exists_list = ["AUTHORS.md", "CONTRIBUTING.md", "CONTRIBUTORS.md", "LICENSE"]
+    check_content_list = check_exists_list.pop(2)
+    check_content_list = check_content_list.append("README.rst")
+
```

Comment:
I'm sure this shouldn't be committed. You have to actively check the style of the README file (either MD or RST)

Reply:
Yes I forgot to delete it :,)